tests/models/embedding/language/test_embedding.py

```diff
@@ -21,15 +23,14 @@
 @pytest.mark.parametrize("model", MODELS)
 @pytest.mark.parametrize("dtype", ["half"])
 def test_models(
-    monkeypatch,
     hf_runner,
     vllm_runner,
     example_prompts,
     model,
     dtype: str,
 ) -> None:
-    if model in ENCODER_ONLY:
-        monkeypatch.setenv("VLLM_ATTENTION_BACKEND", "XFORMERS")
+    if model not in ENCODER_ONLY and current_platform.is_cpu():
+        pytest.skip("Skip large embedding models test on CPU.")
```

Comment:
I only enable encoder-only model test here, because gemma2 is only supported on GPU and e5-mistral-7b might significantly slow down the test.

Reply:
Sounds good to me.